README.md

```diff
@@ -0,0 +1,11 @@
+# tr.gov.uyap.Editor
```

Comment:
This file is unlikely to be ever read by a regular user.

Reply:
Yeah probably no one will ever see that. But pasted manifest description just in case lol